src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -116,6 +116,34 @@ class MessageHelper {
       const hmi_apis::Common_ServiceEvent::eType service_event,
       const hmi_apis::Common_ServiceUpdateReason::eType service_update_reason);
 
+  class ServiceStatusUpdateNotificationBuilder {
```

Comment:
@mked-luxoft add description

Reply:
@LuxoftAKutsan  fixed in 867545c77190a7d6612374fffffafb156c271cdf